<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegularClass</class>
 <widget class="QDialog" name="RegularClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Regular</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>601</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Delete from</string>
      </property>
      <widget class="QTextEdit" name="textEditFrom">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>20</y>
         <width>231</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Delete to</string>
      </property>
      <widget class="QTextEdit" name="textEditTo">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>20</y>
         <width>231</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>591</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Add Files</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="regular.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>RegularClass</receiver>
   <slot>slotProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>RegularClass</receiver>
   <slot>slotPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotProcess()</slot>
  <slot>slotPath()</slot>
  <slot>slotOldTextChanged()</slot>
 </slots>
</ui>
